### ISSUE TITLE
Add acknowledgment to ClimateMachine.jl repository in Documentation / README. 

### ISSUE THREAD

**Issue by jakebolewski** [CliMA/ClimaCore.jl#191]
We should add an acknowledgment to the original ClimateMachine.jl repository (eventually paper with DOI link) and explain the respective differences between ClimateMachine and ClimaCore.

**Comment by valeriabarra** [comment]
I agree. There are many ways in which we can acknowledge recent and past contributions, and contributors. 
We don't have to wait for a paper to have a DOI. For instance, GitHub has a [nice integration](https://guides.github.com/activities/citable-code/) with Zenodo DOIs. We can also add a `CITATION.cff` file so that we can have a "Cite this repository" button on our landing repository GitHub page (see attached example).
<img width="198" alt="GHCite" src="https://user-images.githubusercontent.com/39932030/133676006-c9d2553e-75d3-4e54-bb06-9c5ed3b003b5.PNG">

Also, related to this, I've tried to advocate for a "Project Alumni" section on our [project website](https://clima.caltech.edu/our-team/), even just a list of alumni names, nothin fancy.

**Comment by jakebolewski** [comment]
One of the long term admins is probably the best to add Zendo integration for the organization.  Zendo also needs a release IIRC.

**Comment by valeriabarra** [comment]
Yes, and ClimateMachine.jl has two releases (since we are talking about ways to cite ClimateMachine.jl and acknowledge its contributors)

**Comment by simonbyrne** [comment]
We do have Zenodo hooked up now:
https://zenodo.org/record/5748473

**Comment by valeriabarra** [comment]
Ok, I started two quick PRs, one for ClimateMachine (PR [#2250](https://github.com/CliMA/ClimateMachine.jl/pull/2250)) and one for ClimaCore (PR #329 ) to add Zenodo's DOI badges to the top-level README for now. If we want to add the "Cite this repository" button as well I can look into adding the `CITATION.cff` file
